frontend/src/Dashboard/CrossingListPage/CrossingList.scss

```diff
@@ -2,4 +2,8 @@
 	background-color: #F4F1F6;
 	width: 1280px;
 	border-top: 10px solid #F4F1F6;
-}
\ No newline at end of file
+}
+
+.hiddenCrossing {
+  display: none;
```

Comment:
I made a note above about a11y around display none:
http://a11yproject.com/posts/how-to-hide-content/

We might choose an approach and make a wider purpose utility class to us inside of a `utility.scss` file instead of stuffing this in this file. Or considering using BEM to make it clear how this is specific for the component. Ex: 
```css
.CrossingListItem--hidden {
  ...
}
```